src/tuvx/interface_grid_map.F90

```diff
@@ -6,7 +6,7 @@ module tuvx_interface_grid_map
   use iso_c_binding,       only : c_ptr, c_loc, c_int, c_size_t, c_char
   use tuvx_grid_warehouse, only : grid_warehouse_t
   use tuvx_grid,           only : grid_t
-  use musica_tuvx_util,    only : to_f_string, string_t_c
+  use tuvx_interface_util, only : to_f_string, string_t_c
```

Comment:
It looks like `to_f_string` and `string_t_c` are not used anywhere in the code. Does it make sense to remove them?

Reply:
removed!